src/utils.py

```diff
@@ -7,7 +14,6 @@
 import regex
 import math
 import geocoder
-import dbus
```

Comment:
Unused and will crash if emulated

Reply:
Actually the stop_splash_service call uses this and this is only used on the NHL Scoreboard premade image.  Without the dbus, the access to the systemctl to stop the start up splash animation will not work.  It's part of the code because of how the image is built (it clones the latest version from github)

Reply:
Yup, just clarifying why it's there and why it's used.